crowbar_framework/app/models/service_object.rb

```diff
@@ -1428,6 +1454,35 @@ def apply_role(role, inst, in_queue)
     chef_daemon(:start, chef_daemon_nodes)
   end
 
+  def delete_remove_role_from_runlist(node_name, role_to_remove, inst)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for delete_remove_role_from_runlist is too high. [24.27/15]